.github/workflows/govulncheck.yml

```diff
@@ -0,0 +1,163 @@
+          # Convert the results of type ["vulnKey1", "vulnKey2", ...] in files named like "main", "v1.7", etc to a struct of structure like
+          # {"vulnKey1": ["main", "v1.7", ...], "vulnKey2": ["main", "v1.7", ...]}
+          
+          # Start the script with an empty json object
+          vuln_to_versions="{}"
+          cd results
+          for version in *; do
+            while IFS= read -r vuln;
+            do
+              [[ -z "${vuln}" ]] && continue
+              # Actions from this like:
+              # * is giving the shell var for the vulnerability key to jq (--arg vl "${vuln}") 
+              # * is giving the shell var that contains the scanned version to jq (--arg vers "${version}"
+              # * is getting from vuln_to_versions the key name that needs to be a vulnerability key and assigns to its array the currently proceesed version ('.[$vl] = .[$vl] + [$vers]')
+              # * this is generating outputs like {"vulnKey": ["version1", "version2"]}. Eg: {"GO-2024-2947":["v1.7"]}
+              # * in the end overwrites the content of vuln_to_versions with the newly generated content 
+              vuln_to_versions="$(echo "${vuln_to_versions}" | jq -c --arg vl "${vuln}" --arg vers "${version}" '.[$vl] = .[$vl] + [$vers]')"
+            done <<< "$(cat $version | jq -r '.[]')" # This one is exploding a json array into multiple lines
+          done
+          
+          # This is just using the ".key" that is the vulnerability key and ".value" that is the list of affected version(s)
+          # to generate the commmands to create GitHub issues.
+          while IFS= read -r vuln; 
+          do 
+            vuln_key="$(echo ${vuln} | jq -r '.key')"
+            affected_versions="$(echo ${vuln} | jq -r '.value[]' | xargs)"
+            ticket_title="${vuln_key} reported"
+
+            reported_issues="$(gh issue -R opentofu/opentofu list --search "\"${ticket_title}\" state:open" --json number)"
+            no_of_issues="$(echo ${reported_issues} | jq -r '. | length')"
+            reported_issues="$(echo $reported_issues| jq -r '.[] | .number' | xargs)"
+            [[ ${no_of_issues} -ge 1 ]] && echo "Vulnerabilties found but already reported for ${vuln_key} in: ${reported_issues}" && exit 0
+          
+            echo "This vulnerability is affecting the following versions: ${affected_versions}" > ticket_content
+            echo "" >> ticket_content
+            echo "*Vulnerability info:* https://pkg.go.dev/vuln/${vuln_key}" >> ticket_content
+            echo "*Pipeline run:* ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}" >> ticket_content
+            # gh issue create --repo opentofu/opentofu --title "${ticket_title}" --body-file ticket_content
+            echo "Create issue..."
+            echo "Title: ${ticket_title}"
+            echo "Content:"
+            cat ticket_content
+            echo "---------"
+            echo "---------"
```

Comment:
You can check the [last run](https://github.com/opentofu/opentofu/actions/runs/14105856309/job/39512218182) to see the output.
Any other suggestions of information to be added is more than welcome.